dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESMappingAPIImpl.java

```diff
@@ -388,7 +392,7 @@ protected void loadPermissions(final Contentlet con, final Map<String,Object> m)
 	public static final String elasticSearchDateTimeFormatPattern="yyyy-MM-dd'T'HH:mm:ss";
 	public static final FastDateFormat elasticSearchDateTimeFormat = FastDateFormat.getInstance(elasticSearchDateTimeFormatPattern);
 
-	public static final FastDateFormat timeFormat = FastDateFormat.getInstance("HHmmss");
+	public static final FastDateFormat timeFormat = FastDateFormat.getInstance("HH:mm:ss");
```

Comment:
I am wondering if this should be on the dotmarketing-config with a default on the code